readme.md

```diff
@@ -0,0 +1 @@
+# fix linters errors
```

Comment:
- [ ] Pull Request should have a summary describing introduced changes in more detail, kindly add all the changes to this PR description